### PR TITLE
add --version

### PR DESCRIPTION
**Pull request by MFTabriz**

### CODE REVIEW

File: sphinxlint/__main__.py [hunk 1]
```diff
@@ -109,6 +109,12 @@ def __call__(self, parser, namespace, values, option_string=None):
         help="comma-separated list of fields used to sort errors by. Available "
         f"fields are: {SortField.as_supported_options()}",
     )
+    parser.add_argument(
+        "--version",
+        action="version",
+        version=" ".join(["sphinxlint", __version__]),
+        help="print version",
+    )
```
Comment (hugovk) [inline sphinxlint/__main__.py:117]: Bit more concise:
```suggestion
    parser.add_argument(
        "-V", "--version", action="version", version=f"%(prog)s {__version__}"
    )
```

It takes care of the help text for us:

```console
❯ sphinx-lint -h
usage: sphinx-lint [-h] [-v] [-i IGNORE] [-d DISABLE] [-e ENABLE] [--list] [--max-line-length MAX_LINE_LENGTH] [-s SORT_BY] [-V] [paths ...]

positional arguments:
  paths

options:
  -h, --help            show this help message and exit
  -v, --verbose         verbose (print all checked file names)
  -i IGNORE, --ignore IGNORE
                        ignore subdir or file path
  -d DISABLE, --disable DISABLE
                        comma-separated list of checks to disable. Give "all" to disable them all. Can be used in conjunction with --enable
                        (it's evaluated left-to-right). "--disable all --enable trailing-whitespace" can be used to enable a single check.
  -e ENABLE, --enable ENABLE
                        comma-separated list of checks to enable. Give "all" to enable them all. Can be used in conjunction with --disable
                        (it's evaluated left-to-right). "--enable all --disable trailing-whitespace" can be used to enable all but one check.
  --list                List enabled checkers and exit. Can be used to see which checkers would be used with a given set of --enable and
                        --disable options.
  --max-line-length MAX_LINE_LENGTH
                        Maximum number of characters on a single line.
  -s SORT_BY, --sort-by SORT_BY
                        comma-separated list of fields used to sort errors by. Available fields are: filename,line,error_type
  -V, --version         show program's version number and exit
```

And prints the name without a typo ;)

```console
❯ sphinx-lint --version
sphinx-lint 0.6.7
```

Comment (hugovk) [inline sphinxlint/__main__.py:117]:
```suggestion
    parser.add_argument(
        "-V", "--version", action="version", version=f"Sphinx Lint {__version__}"
    )
```
Or we could write the name as it appears in the README:
```console
❯ sphinx-lint --version
Sphinx Lint 0.6.7
```
This is similar to at least Homebrew:
```console
❯ brew --version
Homebrew 4.1.2
```

**Review by hugovk — COMMENTED** [review]
Thanks for the PR! Here's a couple of alternative suggestions:

File: sphinxlint/__main__.py (hunk 1 above)
Comment (MFTabriz) [reply sphinxlint/__main__.py]: @hugovk thanks for the feedback. I applied the changes but I'm a bit hesitant about having both `-v` and `-V` as flags. What do you think about dropping the short flag (`-V`) and just having `--version`?

**Review by hugovk — APPROVED** [review]

File: sphinxlint/__main__.py (hunk 1 above)
Comment (AA-Turner) [reply sphinxlint/__main__.py]: ``-V`` is a common flag (see e.g. ``python -V``), and the harm of a typo is minimal here, so I'd support keeping it.

A

Comment (hugovk) [reply sphinxlint/__main__.py]: Yep, it's pretty common to use these short flags, for example Python:

https://docs.python.org/3.13/using/cmdline.html#cmdoption-V
https://docs.python.org/3.13/using/cmdline.html#cmdoption-1

And pytest:

https://docs.pytest.org/en/6.2.x/reference.html#id91

**Review by ezio-melotti — APPROVED** [review]

File: sphinxlint/__main__.py (hunk 1 above)
Comment (MFTabriz) [reply sphinxlint/__main__.py]: @hugovk @AA-Turner thanks a lot! I kept the flag.

**Comment by hugovk** [comment]
Thank you!

**Pull request merged**